lib/cocoapods/installer/xcode/pods_project_generator/target_installer.rb

```diff
@@ -89,7 +119,7 @@ def custom_build_settings
           # Creates the directory where to store the support files of the target.
           #
           def create_support_files_dir
-            target.support_files_dir.mkdir
+            target.support_files_dir.mkdir unless File.exist?(target.support_files_dir)
```

Comment:
I just try it and found out that because support files dir not be removed that Pods dir exits(that made mkdir fail).